extensions/amp-facebook-comments/0.1/validator-amp-facebook-comments.protoascii

```diff
@@ -0,0 +1,77 @@
+#
+# Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-facebook-comments
+  html_format: AMP
+  html_format: AMP4ADS
```

Comment:
I can't actually see the build logs, so I pulled the files down and tested locally. Looks like the only remaining validator test is trying to confirm that we really want to allow this for AMP4ADS format. @dvoytenko tells me that we don't currently, so we need to remove this line.